### DISCUSSION TITLE
Installed poetry, but No module named 'poetry'

### DISCUSSION THREAD

**Discussion by jqhr** [pkgxdev/pkgx#878] [Help]
![图片](https://github.com/pkgxdev/pkgx/assets/23305067/1be39a5b-1b70-4042-8d99-0a3e0eee67b5)
What's wrong with it?

**Comment by chidubemOkafor** [comment]
I don't think poetry way installed correctly

**Comment by chidubemOkafor** [comment]
check if it is in your node module

**Comment by mxcl** [comment]
This is fixed. Reinstall poetry.
